webapp/components/post_view/post_info/post_info.jsx

```diff
@@ -392,7 +482,7 @@ export default class PostInfo extends React.Component {
                     {this.createRemovePostButton()}
                 </li>
             );
-        } else if (!isPending) {
+        } else if (!post.failed) {
```

Comment:
Does this change mean that the post dropdown will be displayed when the post is pending and being sent to the server?